src/overlap_check/ProgressBar.hpp

```diff
@@ -0,0 +1,24 @@
+
+#ifndef DAGMC_PROGRESSBAR_H
+#define DAGMC_PROGRESSBAR_H
+
+#include <string>
+
+class ProgressBar {
+
+ public:
+  // constructor
+  ProgressBar();
```

Comment:
Constructor not required as initialization occurs via class member initializers, so this can be removed along with the implementation in the `.cpp` file.